paparazzi-gradle-plugin/src/test/projects/robolectric/src/test/java/app/cash/paparazzi/plugin/test/RobolectricTest.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paparazzi.plugin.test
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class RobolectricTest {
+  @Test
+  fun test() {
+    Robolectric.buildActivity(MainActivity::class.java).use { controller ->
+      controller.setup()
+    }
+  }
```

Comment:
```suggestion
  fun test() = Unit
```

reducing to a minimal test highlights the core issue here.  it's merely the presence of both libraries on the classpath interfering with this runner's bootstrap.